Gruntfile.js

```diff
@@ -1,6 +1,63 @@
 'use strict';
 module.exports = function(grunt) {
+    const parsedIconPicker = 'prod/src/js/iconpicker.js';
+    const tempIconsFile = '.icons.temp';
```

Comment:
This file is not supposed to be commited, can you add it to the `.gitignore` ? thanks

Reply:
Done.